README.md

```diff
@@ -88,40 +94,11 @@ api.iris_enrich('domaintools.com').response() == { ... }
 
 You can directly get the html, xml, or json version of the response by calling `.(html|xml|json)()` These only work with non AsyncResults:
 ```python
-html = str(api.domain_search('google').json())
+json = str(api.domain_search('google').json())
 xml = str(api.domain_search('google').xml())
 html = str(api.domain_search('google').html())
 ```
 
-Besides the Iris Investigate method letting you lookup domains, it can also be used to search for domains sharing common attributes. For instance, you can pass it an IP to bring back all domains sharing that IP:
```

Comment:
Took out this. I think examples and sending them to the API docs is a better place for stuff like this.